src/gui/transferlistmodel.cpp

```diff
@@ -526,6 +530,8 @@ QVariant TransferListModel::internalValue(const BitTorrent::Torrent *torrent, co
         return torrent->nextAnnounce();
     case TR_PRIVATE:
         return (torrent->hasMetadata() ? torrent->isPrivate() : QVariant());
+    case TR_PERCENT_SELECTED:
+        return torrent->hasMetadata() ? (static_cast<qreal>(torrent->wantedSize()) * 100) / torrent->totalSize() : 0;
```

Comment:
Anyone knows if there's a torrent creator out there that can create 0 total size torrents? (div by zero) libtorrent doesn't allow it.

Reply:
We don't know torrent size until metadata received.

Reply:
> Anyone knows if there's a torrent creator out there that can create 0 total size torrents? (div by zero) libtorrent doesn't allow it.

I think it doesn't matter to us whether some software allows to create such torrents, but whether such a torrent is considered valid by `libtorrent`. I believe No.